### PR TITLE
modification for workbench version 1.6.0

### PR DESCRIPTION
**Pull request by tomaskrcka**
Just modifications regarding to new version of xdk workbench - 1.6.0

### CODE REVIEW

File: src/MQTTXDK.c [hunk 1]
```diff
@@ -120,35 +123,33 @@ int ConnectNetwork(Network* net, int8_t* addr, int port)
...
         sl_Close(net->my_socket);
         return retVal;
     }
 
-    sysTickIntHandle = OS_timerCreate((const int8_t *) "MqttTimer",
+    sysTickIntHandle = xTimerCreate((const char *) "MqttTimer",
                                       TIMER_TICK_PERIOD,
-                                      OS_AUTORELOAD_ON,
-                                      NULL,
+									  pdTRUE,
```
Comment (jamesmunns) [inline src/MQTTXDK.c]: Tabs vs spaces here

File: src/tasks.c [hunk 2]
```diff
@@ -106,24 +109,26 @@ void Restart(void)
     printf("WiFi deinit = %d\n\r", WiFiDeinit());
     wifiConnected = false;
     MqttConnectInit();
-    OS_taskCreate(WifiReconnect,
+    xTaskCreate(WifiReconnect,
                   (const int8_t *) "WiFi Reconnection",
                   TASK_STACK_SIZE,
+				  NULL,
```
Comment (jamesmunns) [inline src/tasks.c]: More tabs

File: src/tasks.c [hunk 3]
```diff
@@ -140,9 +145,10 @@ void WifiConnectInit(void)
     if(-1 == WiFiInit(&WifiConnectionCallback))
     {
         printf("Connection to WLAN failed. Retrying in %d ms\n", (int)RECONNECT_PERIOD);
-        OS_taskCreate(WifiReconnect,
+        xTaskCreate(WifiReconnect,
                       (const int8_t *) "WiFi Reconnection",
                       TASK_STACK_SIZE,
+					  NULL,
```
Comment (jamesmunns) [inline src/tasks.c]: Tabs here and the next few changes

**Comment by chainq** [comment]
👍

**Pull request merged**